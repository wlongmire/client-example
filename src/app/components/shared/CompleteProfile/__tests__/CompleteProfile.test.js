/* eslint-env node, jest */

import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { CompleteProfile } from '../index'

jest.mock('react-router', () => ({ browserHistory: [] }))

describe('>>> Set Password Component Snapshot', () => {
  it('+++capturing Snapshot with', () => {
    const renderedValue = shallow(
      <CompleteProfile
        goToNextStep={jest.fn()}
      />)
    expect(toJson(renderedValue)).toMatchSnapshot()
  })
})