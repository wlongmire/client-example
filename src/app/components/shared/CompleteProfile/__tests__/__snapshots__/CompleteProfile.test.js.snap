// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>> Set Password Component Snapshot +++capturing Snapshot with 1`] = `
<div
  className="completeProfile"
>
  <form>
    <FormGroup
      bsClass="form-group"
      controlId="firstName"
    >
      <ControlLabel
        bsClass="control-label"
        srOnly={false}
      >
        First Name
      </ControlLabel>
      <FormControl
        bsClass="form-control"
        componentClass="input"
        id="firstName"
        label="Text"
        onChange={[Function]}
        type="text"
      />
      <HelpBlock
        bsClass="help-block"
        className="helpBlockRed"
      >
        *Required
      </HelpBlock>
    </FormGroup>
    <FormGroup
      bsClass="form-group"
      controlId="lastName"
    >
      <ControlLabel
        bsClass="control-label"
        srOnly={false}
      >
        Last Name
      </ControlLabel>
      <FormControl
        bsClass="form-control"
        componentClass="input"
        id="lastName"
        label="Text"
        onChange={[Function]}
        type="text"
      />
      <HelpBlock
        bsClass="help-block"
        className="helpBlockRed"
      >
        *Required
      </HelpBlock>
    </FormGroup>
    <FormGroup
      bsClass="form-group"
      controlId="jobTitle"
    >
      <ControlLabel
        bsClass="control-label"
        srOnly={false}
      >
        Job Title
      </ControlLabel>
      <FormControl
        bsClass="form-control"
        componentClass="input"
        id="jobTitle"
        label="Text"
        onChange={[Function]}
        placeholder="optional"
        type="text"
      />
      <div
        className="completeSpace"
      />
    </FormGroup>
    <Row
      bsClass="row"
      componentClass="div"
    >
      <Col
        bsClass="col"
        componentClass="div"
        lg={8}
        md={8}
        sm={8}
        xs={8}
      >
        <FormGroup
          bsClass="form-group"
          controlId="phone"
        >
          <ControlLabel
            bsClass="control-label"
            srOnly={false}
          >
            Phone Number
          </ControlLabel>
          <FormControl
            bsClass="form-control"
            componentClass="input"
            id="phone"
            label="Text"
            onChange={[Function]}
            type="text"
          />
          <HelpBlock
            bsClass="help-block"
            className="helpBlockRed"
          >
            *Required
          </HelpBlock>
        </FormGroup>
      </Col>
      <Col
        bsClass="col"
        componentClass="div"
        lg={4}
        md={4}
        sm={4}
        xs={4}
      >
        <FormGroup
          bsClass="form-group"
          controlId="phoneExt"
        >
          <ControlLabel
            bsClass="control-label"
            srOnly={false}
          >
            Ext
          </ControlLabel>
          <FormControl
            bsClass="form-control"
            componentClass="input"
            id="phoneExt"
            label="Text"
            onChange={[Function]}
            placeholder="optional"
            type="text"
          />
        </FormGroup>
      </Col>
    </Row>
  </form>
  <div
    className="completeProfileButton"
  >
    <Button
      active={false}
      block={false}
      bsClass="btn"
      bsStyle="default"
      disabled={false}
      onClick={[Function]}
    >
      Complete Profile
    </Button>
    <div
      className="completeSpace"
    />
  </div>
</div>
`;
